pkg/cosign/attestation/attestation.go

```diff
@@ -40,6 +43,37 @@ type CosignPredicate struct {
 	Timestamp string
 }
 
+// VulnPredicate specifies the format of the Vulnerability Scan Predicate
+type CosignVulnPredicate struct {
+	Invocation Invocation `json:"invocation"`
+	Scanners   []Scanner  `json:"scanners"`
+	Metadata   Metadata   `json:"metadata"`
+}
+
+type Invocation struct {
```

Comment:
We should probably also add a README in here under specs/ that describes all of these fields in detail and how they should be used.

Reply:
Exactly! This is the next that we are going to do🤝